client/src/pages/Calendar.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from "react";
+import Alert from "react-bootstrap/Alert";
+
+const Calendar = () => {
+	const [bookingsData, setBookingsData] = useState([]);
+	const [error, setError] = useState(null);
+	const [isLoading, setIsLoading] = useState(false);
+
+	useEffect(() => {
+		setIsLoading(true);
+		fetch("/api/bookings?filter=2weeks")
+			.then((response) => {
+				if (!response.ok) {
+					throw Error(
+						"There was an error getting the calendar data, Please try again"
+					);
+				}
+				return response.json();
+			})
+			.then((data) => {
+				setBookingsData(data);
+				setError(null);
+			})
+			.catch((err) => {
+				setError(err.message);
+			})
+			.finally(() => {
+				setIsLoading(false);
+			});
+	}, []);
+
+	if (isLoading) {
+		return (
+			<div className="d-flex justify-content-center">
+				<div className="spinner-grow">
+					<span className="visually-hidden">Loading...</span>
+				</div>
+			</div>
+		);
+	}
+	if (error != null) {
+		return <Alert variant="danger">{error}</Alert>;
+	}
+
+	const pickUpDates = [];
+
+	bookingsData.forEach((client) => {
+		let found = pickUpDates.find((el) => el.date == client.pick_up_date);
+		if (found) {
+			found.clients.push(client);
+		} else {
+			pickUpDates.push({ date: client.pick_up_date, clients: [client] });
+		}
+	});
+
+	return (
+		<div className="container py-5">
+			<div className="p-4  bg-dark rounded-3 no-border">
+				{" "}
+				<h2 className="text-light bg-dark">Bike pick-up calendar</h2>
+			</div>
```

Comment:
```suggestion
			<h1 className="my-4">Bike pick-up calendar</h1>
```
Let's use a plain heading. We should also make this an `h1` as it is the top level heading of the page

Reply:
I got it.